packages/node/src/methods/app-instance/propose-install/controller.ts

```diff
@@ -66,17 +67,12 @@ export default class ProposeInstallController extends NodeController {
 
     const myIdentifier = publicIdentifier;
 
-    const multisigAddress = await store.getMultisigAddressFromOwnersHash(
-      hashOfOrderedPublicIdentifiers([myIdentifier, proposedToIdentifier])
+    const multisigAddress = getCreate2MultisigAddress(
+      [myIdentifier, proposedToIdentifier],
+      networkContext.ProxyFactory,
+      networkContext.MinimumViableMultisig
     );
 
-    // TODO: DRY this at top level of most calls that query a channel
```

Comment:
Make an issue to address this and leave it here until the issue implements this better. Removing this makes the software possibly more error-prone, not less.